ansible/roles/media-licenses/templates/vidis-media-activation-data-sync-cronjob-configmap.yml.j2

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ NAMESPACE }}
+  name: vidis-media-activation-data-sync-cronjob-configmap
```

name to long don't write names like i-lvoe-to-dance-with-an-dragon-at-night
```suggestion
  name: vidis-sync-cronjob-configmap
```